docs/integration/jenkins.md

```diff
@@ -0,0 +1,58 @@
+**jenkins.http.response.code.other** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a non-informational status code that is not in the list - HTTP/200, HTTP/201, HTTP/204, HTTP/304, HTTP/400, HTTP/403, HTTP/404, HTTP/500, or HTTP/503 
+**jenkins.nodes.builds** <br>*(long counter)*      |  The count of build nodes available to Jenkins
+**jenkins.nodes.builds.time** <br>*(long counter)* *(ms)*      |  The time nodes spend for building
+**jenkins.nodes.offline** <br>*(long gauge)*      |  The count of build nodes available to Jenkins but currently off-line
+**jenkins.nodes.online** <br>*(long gauge)*      |  The count of build nodes available to Jenkins and currently on-line
+**jenkins.queue.blocked** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.buildable** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.pending** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.size** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue
+**jenkins.queue.stuck** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.plugins.active** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that started successfully
+**jenkins.plugins.with_update** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that have an newer version reported as available in the current Jenkins update center metadata held by Jenkins. This value is not indicative of an issue with Jenkins but high values can be used as a trigger to review the plugins with updates with a view to seeing whether those updates potentially contain fixes for issues that could be affecting your Jenkins instance
+**jenkins.plugins.inactive** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that are not currently enabled
+**jenkins.plugins.failed** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that failed to start. A value other than 0 is typically indicative of a potential issue within the Jenkins installation that will either be solved by explicitly disabling the plugin(s) or by resolving the plugin dependency issues
+**jenkins.executors.free** <br>*(long gauge)*      |  The count of executors available to Jenkins that are not currently in use
+**jenkins.executors.in_use** <br>*(long gauge)*      |  The count of executors available to Jenkins that are currently in use
+**jenkins.runs.success** <br>*(long counter)*      |  The count of job runs which performed succeed
```

Comment:
The count of job runs which performed succeed
=>
The count of job runs which performed successfully

Reply:
done